runtime/src/iree/hal/drivers/vulkan/cts/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+unset(ADDITIONAL_TEST_ARGS)
+if(ANDROID)
+  # Disable this test due to flaky failures on Google Pixel 6 Pro.
+  set(ADDITIONAL_TEST_ARGS "--gtest_filter=-*WaitForFiniteTime*")
```

Comment:
You can also (by convention) prefix private variables with an underscore, like `_ADDITIONAL_TEST_ARGS`. There is the expectation there that using such underscore-prefixed variables outside of the file they are defined is unsupported. Using `unset` (or set to empty string/list) is still good practice here though, in case the variable happened to be set by another module.

Reply:
According to the [doc](https://cmake.org/cmake/help/latest/manual/cmake-variables.7.html):

--------

Note CMake reserves identifiers that:
begin with CMAKE_ (upper-, lower-, or mixed-case), or

begin with _CMAKE_ (upper-, lower-, or mixed-case), or

begin with _ followed by the name of any [CMake Command](https://cmake.org/cmake/help/latest/manual/cmake-commands.7.html#manual:cmake-commands(7)).

--------

Since you don't know what will be added to the CMake commands in the future the actual set of names you should avoid is all names prefixed with `_`.

Reply:
Oh, I haven't seen that argument before. Interesting. I have seen many other projects using underscore by convention though 🤔. If we want to be extra careful we could prefix with `_IREE`.

I'm fine with either way here.